src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -57,14 +57,25 @@ public enum LanguageVersion
         /// </list> 
         /// </remarks> 
         CSharp6 = 6,
+
         /// <summary>
         /// C# language version 7.
         /// </summary>
         CSharp7 = 7,
+
+        /// <summary>
+        /// The latest version of the language supported.
+        /// </summary>
+        Latest = int.MaxValue,
```

Comment:
@AdamSpeight2008 
> You still haven't quantified the difference between Latest and Default.

`/langver:default` gives you whatever language version is the default for the version of the compiler you're using.
We are not introducing a "Default" value for the enum.
We are not introducing a command-line flag that means the latest supported version, whatever that is.
"Latest" is a value you can use for the enum in the API to get the latest supported version of the language.

I hope that answers your questions.

---
In reply to: [72684514](https://github.com/dotnet/roslyn/pull/12796#discussion_r72684514) [](ancestors = 72684514)